assets/js/common/Tags/Tags.jsx

```diff
@@ -15,13 +16,33 @@ const tagValidationDefaultMessage = (
   </>
 );
 
+function TagDeleteButton({ onClick, disabled }) {
+  return (
+    <span
+      aria-hidden="true"
+      className={classNames('cursor-pointer group-hover:opacity-60', {
```

Comment:
Are the classes correct?
Originally the `ml-2` was on the `span`, but now you have moved it the icon.
Maybe it doesn't change anything, but I got surprised

Reply:
This is a styling issue because the tooltip need a wrap (otherwise the refs of the library went wrong and the tooltip component crash), so I moved to the icon itself